runtime.md

```diff
@@ -29,24 +29,59 @@ The root directory to the bundle is provided in the state so that consumers can
 }
 ```
 
-## Lifecycle
+## Typical lifecycle
+
+A typical lifecyle progresses like this:
+
+1. There is no container or running application
+2. A user tells the runtime to start a container+application
```

Comment:
`A user tells the runtime to start a container`  Don't use the word application, its just a container